lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -21,10 +21,7 @@
         </p>
     % if not credit_status["purchased"] and not credit_status["error"] :
         <p class="message-copy credit-eligibility-msg">
-            ${_("You are now eligible for credit. {congrats}").format(
-                    congrats="<b>Congratulations</b>",
-                )
-            }
+            ${_("You are now eligible for credit. <b>Congratulations!</b>")}
```

Comment:
Do we also need to use placeholders for html tags inside a string for translation e.g., `<b></b>`, `<p></p>`?
Like we did for anchor tag `{link_start}{link_end}`?